asset-transfer-basic/application-javascript/app.js

```diff
@@ -41,10 +41,10 @@ async function main() {
         // Steps:
```

Comment:
Do you mind enhancing the func at line 18, as below, for cases when input is empty.

```
function prettyJSONString(inputString) {
    if(inputString) {
        return JSON.stringify(JSON.parse(inputString),null,2);
    }
    else {
        return inputString;
    }
}
```